docs/searchpredicates.txt

```diff
@@ -110,7 +110,14 @@ Geoshape.circle(37.97, 23.72, 50)
 Geoshape.box(37.97, 23.72, 38.97, 24.72)
 //WKT (requires JTS for polygon, linestring and multi-point/line/polygon geometries)
```

Comment:
Change to "(requires JTS)" and annotate individually below. Without JTS I was able to create MultiPoint/MultiLine, though would indexing these work without JTS?